rules/windows/powershell/powershell_script/posh_ps_add_dnsclient_rule.yml

```diff
@@ -0,0 +1,26 @@
+title: Windows DNS Client Commands
```

Comment:
I think the title is too generic agains the rule detection.
Somethink like "Powershell Add Name Resolution Policy Table rule" ?